test/support/video_filter_helpers.ex

```diff
@@ -0,0 +1,46 @@
+defmodule VideoFilter.Helpers do
+  @moduledoc false
+  import ExUnit.Assertions
+
+  require Membrane.Logger
+
+  @spec prepare_paths(binary(), binary(), binary()) :: {binary(), binary(), binary()}
+  def prepare_paths(input_file, ref_file, tmp_dir) do
+    in_path = "../fixtures/#{input_file}" |> Path.expand(__DIR__)
+    ref_path = "../fixtures/#{ref_file}" |> Path.expand(__DIR__)
+    out_path = Path.join(tmp_dir, ref_file)
+    {in_path, out_path, ref_path}
+  end
+
+  @spec create_ffmpeg_reference(binary, binary, binary) :: nil | :ok
+  def create_ffmpeg_reference(input_file, output_reference_path, filter_descr) do
+    full_input_path = "../fixtures/#{input_file}" |> Path.expand(__DIR__)
+
+    {result, exit_status} =
+      System.cmd(
+        "ffmpeg",
+        [
+          # overrides the output file without asking if it already exists
+          "-y",
+          "-i",
+          full_input_path,
+          "-vf",
+          filter_descr,
+          output_reference_path
+        ],
+        stderr_to_stdout: true
+      )
+
+    if exit_status != 0 do
+      Membrane.Logger.error("ffmpeg command error")
+      Membrane.Logger.error(result)
```

Comment:
raise here, otherwise the test won't stop and with capure_log you might not see these logs